src/utils/install/journal_defaults.json

```diff
@@ -2938,5 +2938,20 @@
         "value": {
             "default": "1"
         }
+    },
+    {
+        "group": {
+            "name": "general"
+        },
+        "setting": {
+            "description": "If enabled, authors will see detailed (anonymous) information about peer reviews inc. due and completed dates on the article information page.",
```

Comment:
Hmm, it might not be clear what happens when this is disabled: 
"If enabled, authors will see the requested, due, and complete date for their papers' review assignment. Otherwise, they will only have access to the review comments"


I would avoid using the word "anonymous" since it it won't be true for open reviews.